src/features/sanctions/hooks/useOfacSanctioned.ts

```diff
@@ -0,0 +1,24 @@
+import { useQuery } from '@tanstack/react-query';
+import { isAddress, isAddressEqual } from 'viem';
+
+import { useEvmAccount } from '../../wallet/hooks/evm';
+
+const OFAC_SANCTIONED_ADDRESSES_ENDPOINT =
+  'https://raw.githubusercontent.com/0xB10C/ofac-sanctioned-digital-currency-addresses/lists/sanctioned_addresses_ETH.json';
+
+export const useOfacSanctioned = () => {
+  const evmAddress = useEvmAccount().addresses[0]?.address;
+
+  const sanctionedAddresses = useQuery<string[]>({
+    queryKey: ['ofac sanctioned', evmAddress],
```

Comment:
This key is fine but a convention I've used elsewhere in the app is to use the hook name, so 'useOfacSanctioned'